language/tests/fstar/QuestionMark.fst

```diff
@@ -1,162 +0,0 @@
-module QuestionMark
```

Comment:
If you mean removing these files, I agree. We should remove all generated files here from the repo and re-generate them on CI when they are needed.